api/index.js

```diff
@@ -286,6 +287,9 @@ const limiter = rateLimit({
 
 app.use("/api/v1/faucet", limiter, faucet.faucet);
 
+app.get("/api/v1/cere-tokens/total-supply", ethNetwork.getCereTotalSupply);
```

Comment:
Rename to `/api/v1/cere-erc20-token/total-supply`
Please, use the same pattern for the second route

Reply:
Updated